RadarSDK/RadarSettings.m

```diff
@@ -226,6 +230,32 @@ + (void)setFeatureSettings:(RadarFeatureSettings *)featureSettings {
     }
 }
 
++ (void)updateSdkConfigurationFromServer {
+    [[RadarAPIClient sharedInstance] getConfigForUsage:@"sdkConfigUpdate"
+                                              verified:NO
+                                     completionHandler:^(RadarStatus status, RadarConfig *config) {
+                                         if (status != RadarStatusSuccess || !config) {
+                                            return;
+                                         }
+                                         [RadarSettings setSdkConfiguration:config.meta.sdkConfiguration];
+                                     }];
+}
```

Comment:
We don't call the API client anywhere else in `RadarSettings` — I think we can find a better home for this call.

I'd also say that the name `updateSdkConfigurationFromServer` is a clue that isn't correctly placed (I don't think we a want function named `update_____FromServer`)